lib/Elevate/Blockers/IsContainer.pm

```diff
@@ -17,7 +17,9 @@ use Log::Log4perl qw(:easy);
 
 sub check ($self) {    # $self is a cpev object here
     if ( _is_container_envtype() ) {
-        return $self->has_blocker("cPanel thinks that this is a container-like environment, which this script cannot support at this time.");
+
+        return 0 if !$self->should_run_leapp;
```

Comment:
this feels like a good use for unless I'm having trouble seeing `!$`

Reply:
fixed, yeap used it in other locations and forgot to change this one